### DISCUSSION TITLE
Using cbc stand alone executable in a snap

### DISCUSSION THREAD

**Discussion by uenus** [coin-or/Cbc#619] [Discussion]
I have an optimization program which i need to use in a core20 base. I cant find the suitable standalone exe to add in the bin folder. Could some one help me with this.

**Comment by jhmgoossens** [comment]
I’m not a user of core20, but can you use the binaries for ubuntu20 that are published?
Otherwise, you can build your own using coinbrew as described in the [README.md](https://github.com/coin-or/Cbc/blob/master/README.md)
Does this help?

**Reply by uenus** [reply]
Yes I can use the ubuntu20 Releases for amd64 Architecture.  But I couldnot find the binaries for aarch64(arm64) architecture in the binaries. So i will try building them.
